### PR TITLE
Fix some existing pronto errors

### PR DESCRIPTION
**Pull request by simozhan**
Fix some pronto errors like:

1. examples/local_pdc_dev.rb:15:9:
C: Style/YodaCondition: Reverse the order of the operands __FILE__ == $PROGRAM_NAME.
(https://en.wikipedia.org/wiki/Yoda_conditions) main if __FILE__ == $PROGRAM_NAME

2. examples/logger.rb:8:3:
C: Style/EmptyMethod: Put empty method definitions on a single line.
(https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
def debug(*args) ...

3. spec/pdc/resource/path_spec.rb:16:18:
C: Style/SymbolArray: Use %i or %I for an array of symbols.
(https://github.com/bbatsov/ruby-style-guide#percent-i)
assert_equal [:user_id, :id], path.variables

4. spec/pdc/v1/release_spec.rb:31:7:
C: Style/CommentedKeyword: Do not place comments on the same line as the end keyword.
end #  count
...

### CODE REVIEW

File: pdc.gemspec [hunk 1]
```diff
@@ -16,24 +16,22 @@ Gem::Specification.new do |s|
...
+  raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.' \
+      unless s.respond_to?(:metadata)
 
+  s.metadata['allowed_push_host'] = 'https://rubygems.org'
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_dependency 'activesupport', '~> 3.2.22.2'
```
Comment (romanofski) [inline pdc.gemspec]: Are these version changes part of the styling changes?

Comment (simozhan) [reply pdc.gemspec]: Yeah. I think it is styling change. It only request the user to add dependency in alphabet order

Comment (romanofski) [reply pdc.gemspec]: True. Sorry my mistake

Comment (simozhan) [reply pdc.gemspec]: np at all. Thanks for the quick response

**Pull request merged**